### ISSUE TITLE
Weird icons on the status

### ISSUE THREAD

**Issue by carks32** [Harmon758/mb_DiscordRichPresence#1]
https://i.imgur.com/gcwuwMV.png

I'm not sure it should be like that, I'm on Windows 7 32bit

**Comment by Harmon758** [comment]
Just to confirm, is the song title, "Barca quemada"?
Also, are you using the latest release, v1.0.4?

**Comment by carks32** [comment]
Yeah, the song is this [one](https://www.last.fm/music/Tri%C3%A1ngulo+de+Amor+Bizarro/_/Barca+Quemada). 

Also, I did download the latest release but in the info appears [this](https://i.imgur.com/zOEGc5M.png), so I don't know what's happening.

**Comment by MichaelGatesDev** [comment]
I am unable to reproduce this issue. Can you use a program like MP3Tag and post a screenshot of the tags here? Most characters are supported already, so my guess is that it's some unused ascii character causing the problem.

**Comment by MichaelGatesDev** [comment]
It appears this is a bug with the UTF16->UTF8 conversion algorithm. In my fork, I have switched to the other suggested method of doing so (where the author got the code from) which has worked for me well thus far.
